libcodechecker/log/option_parser.py

```diff
@@ -3,302 +3,194 @@
 #   This file is distributed under the University of Illinois Open Source
 #   License. See LICENSE.TXT for details.
 # -------------------------------------------------------------------------
-"""
-An extended lookup table for compiler arguments based on ScanBuild
-clang-analyzer.llvm.org/scan-build.html
 
-Used to filter out or change compiler argument not supported by clang
-or clang-tidy.
-
-Keys are the option name, value is the number of options to skip.
-
-Possible improvements:
-    - modular option handling system configuring possibility from config file.
-
-"""
-from __future__ import print_function
-from __future__ import division
-from __future__ import absolute_import
```

Comment:
Why did we remove these imports?

Reply:
This whole file has been removed.

Reply:
True. I keep `print_function` and `absolute_import`. The `division` is not necessary.